projects/ndpi/build.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash -eu
+
+cd /
+tar -xvzf libpcap-1.9.1.tar.gz
+
+export AFL_LLVM_LAF_ALL=1
+
+#Build targets for libfuzzer
+echo "Build targets for libfuzzer."
+
+export CC="clang"
+export CFLAGS="-g -fsanitize=fuzzer-no-link,address,bounds,integer,undefined,null,float-divide-by-zero"
+TARGET_CFLAGS="-g -fsanitize=fuzzer,address,bounds,integer,undefined,null,float-divide-by-zero"
+INCLUDE_DIR="../src/include"
```

Comment:
Set absolute path: lcov + genhtml will fail to display sources for parent directories.